lib/util/utils.ts

```diff
@@ -779,12 +791,13 @@ export function allowNullType<T extends Entity>(
       const savedEntity = entity
       // handling nullable parameters
       if (savedEntity.in) {
-        entity.anyOf = [{ type: entity.type }, { type: "null" }]
+        const typeNull: SchemaObject = setSchemaInfo({ type: "null" }, info)
```

Comment:
what problem was this causing? and how it the new code added here fixing it?

Reply:
generated types should have source map (SchemaInfo). Otherwise, z-schema and yasway will not return source information.